helper/certutil/types.go

```diff
@@ -45,6 +45,16 @@ const (
 	TLSClient
 )
 
+//BlockType indicates the serialization format of the key
+type BlockType string
+
+//Well-known formats
+const (
+	PKCS1Block BlockType = "RSA PRIVATE KEY"
+	PKCS8Block BlockType = "PRIVATE KEY"
+	ECBlock    BlockType = "EC PRIVATE KEY"
+)
```

Comment:
Oops, you're right -- EC format is lookalike but not actually PKCS1. Thanks for the catch.